src/styles/styles.scss

```diff
@@ -0,0 +1,97 @@
+@import './globals.scss';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  z-index: 1500;
+  height: 75px;
+  width: 100%;
+  background-color: $secondaryColor;
+
+  h4 {
+    color: white;
+    font-size: 2rem;
```

Comment:
I understand 1rem = 16px. Maybe we should stick to one

Reply:
Like 1rem for the header

Reply:
Sorry, didn't convey myself properly. I meant to use 32px instead of 2rem. Or define rules where we use rems where px